frontend/libs/studio-components/src/components/StudioCenter/StudioCenter.tsx

```diff
@@ -0,0 +1,15 @@
+import React, { HTMLAttributes, forwardRef } from 'react';
+import classes from './StudioCenter.module.css';
+import cn from 'classnames';
+
+/**
+ * @component
+ *    Component that centers its content both vertically and horizontally.
+ */
+export const StudioCenter = forwardRef<HTMLDivElement, HTMLAttributes<HTMLDivElement>>(
+  ({ className, ...rest }, ref): JSX.Element => (
+    <div ref={ref} className={cn(className, classes.root)} {...rest} />
+  ),
```

Comment:
This is not a big deal, but by declaring the props within its own type, we follows the same pattern as the other components. 

```suggestion
type StudioCenterProps =  HTMLAttributes<HTMLDivElement>
export const StudioCenter = forwardRef<HTMLDivElement, StudioCenterProps>(
  ({ ...rest }, ref): JSX.Element => (
    <div ref={ref} className={cn(rest.className, classes.root)} {...rest} />
  ),
```

Reply:
I agree! 😄